ui/app/mainui/AppMain.qml

```diff
@@ -503,31 +505,22 @@ Item {
 
                     ModuleWarning {
                         readonly property bool isConnected: bannersLayout.isConnected
-
                         objectName: "connectionInfoBanner"
                         Layout.fillWidth: true
-                        text: isConnected ? qsTr("Connected") : qsTr("Disconnected")
```

Comment:
If I understand correctly, "Connected" is also needed, when we go back online after being some time offline.